src/nfv_test_api/v2/controllers/gnodeb.py

```diff
@@ -127,6 +128,33 @@ def get(self, nci: str):
 
         return self.gnb_service.get_one(nci).json_dict(exclude_none=True), HTTPStatus.OK
 
+    @namespace.expect(gnodeb_update_model)
+    @namespace.response(
+        HTTPStatus.OK.value, "The gNodeB config has been updated", gnodeb_model
+    )
+    @namespace.response(
+        HTTPStatus.NOT_FOUND.value, "Couldn't find any gNodeB with given nci"
+    )
```

Comment:
The whole point of put is that you can use it whether something is there or not.  It is idempotent, you just the resource there, regardless of what might or might not be there

Reply:
Okay should I pass the nci as argument as well ? Or do I move the put in the same class as the post?